### PR TITLE
biopsy date UI changes

### PR DESCRIPTION
**Pull request by achen2401**
address this story:
https://www.pivotaltracker.com/story/show/147777315
- remove datepicker from biopsy date field
- add month, day year fields for the biopsy date
- css fixes

### CODE REVIEW

File: portal/static/js/main.js [hunk 1]
```diff
@@ -471,8 +476,13 @@ var fillContent = {
                         if (clinicalValue == "true") {
                             if (hasValue(val.content.issued)) {
                                 var issuedDate = "";
-                                //d M y format
-                                $("#biopsyDate").val(tnthDates.formatDateString(val.content.issued));
+                                //$("#biopsyDate").val(tnthDates.formatDateString(val.content.issued));
+                                var dString = (val.content.issued).substring(0, (val.content.issued).indexOf("T"));
```
Comment (ivan-c) [inline portal/static/js/main.js]: I think these dates are all ISO 8601, do you have a common function for parsing these?

File: portal/static/js/main.js [hunk 2]
```diff
@@ -2909,34 +2920,36 @@ $(document).ready(function() {
...
                 // Check to see if this is a real date
                 var date = new Date(y,m-1,d);
                 if (!(date.getFullYear() == y && (date.getMonth() + 1) == m && date.getDate() == d)) {
-                    $("#errorbirthday").html("Invalid date. Please try again.").show();
+                    errorField.html("Invalid date. Please try again.").show();
                     return false;
                 }
                 else if (date.setHours(0,0,0,0) >= today.setHours(0,0,0,0)) {
-                    $("#errorbirthday").html("Birthday must not be in the future. Please try again.").show();
+                    errorField.html("Birthday must not be in the future. Please try again.").show();
```
Comment (ivan-c) [inline portal/static/js/main.js]: Should these error messages still reference "Birthday" if it's a more generic date validation function?

File: portal/static/js/main.js [hunk 3]
```diff
@@ -386,8 +386,13 @@ var fillViews = {
             var biopsyDate = $("#biopsyDate").val();
             if (a == "true" && hasValue(biopsyDate)) {
                 //note, biopsy date is formatted as mm/dd/yyyy
-                var cDate = new Date(biopsyDate);
-                displayDate = tnthDates.displayDateString(cDate.getMonth()+1, cDate.getDate(), cDate.getFullYear())
+                //var cDate = new Date(biopsyDate);
```
Comment (ivan-c) [inline portal/static/js/main.js]: Please delete old/debug code if unnecessary :)

File: portal/templates/initial_queries_macros.html [hunk 4]
```diff
@@ -362,6 +388,8 @@ <h4 class="modal-title" id="termsReminderModalLabel">{{ _("Please print and fill
         <br/>
       </div>
       <script>
+        /****************** old date picker function ***********/
```
Comment (ivan-c) [inline portal/templates/initial_queries_macros.html]: Please delete old/debug code if unnecessary :)

**Comment by achen2401** [comment]
@ivan-c 
Ivan, thank you, I think I addressed the issues you raised.  Let me know if you see any more problem.

**Comment by ivan-c** [comment]
Thanks! Will merge once tests pass

**Pull request merged**